server/session/world.go

```diff
@@ -915,9 +917,12 @@ func (s *Session) ViewEntityState(e world.Entity) {
 }
 
 // ViewEntityAnimation ...
-func (s *Session) ViewEntityAnimation(e world.Entity, animationName string) {
+func (s *Session) ViewEntityAnimation(e world.Entity, animation animation.Animation) {
```

Comment:
I'd just rename this variable to a to avoid conflicting with the package, same thing in the interface yupe in viewer.go